### PR TITLE
Add Docstring

### LINKED ISSUE #10: Add docstrings to functions missing docstrings
This makes using something like sphinx autodocs easier and using ides like python to create bots a lot faster.

### PR DESCRIPTION
**Pull request by percygautam**
Fix #10

### CODE REVIEW

**Comment by percygautam** [comment]
@TAG-Epic I have added docstring as I understood the functions. Please point out the mistakes and also help me better understand the codebase.

File: speedcord/values.py [hunk 1]
```diff
@@ -4,4 +4,4 @@
 Don't touch version, it is automatically generated.
 """
 
-version = "0.1.57"
\ No newline at end of file
+version = "0.1.57"
```
Comment (TAG-Epic) [inline speedcord/values.py:7]: This should not be included in this pr. The pr this was included in was rejected

File: speedcord/dispatcher.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 
 class OpcodeDispatcher:
+    """Opcode dispatcher for discord API"""
```
Comment (TAG-Epic) [inline speedcord/dispatcher.py:10]: All the docstrings have not provided any value. Please follow examples such as [this](https://github.com/TAG-Epic/speedcord/blob/9d1b07ca5b6400930c796adba209b0a9ec48d6be/speedcord/client.py#L18)

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: speedcord/dispatcher.py (hunk 2 above)
Comment (percygautam) [reply speedcord/dispatcher.py:10]: Okay I'll also add the params

Comment (TAG-Epic) [reply speedcord/dispatcher.py:10]: And please make the docstrings provide something that isn't in the class name or in the .pyi typings

**Comment by percygautam** [comment]
I was working on this. Should I create another PR? @TAG-Epic

**Comment by TAG-Epic** [comment]
Yeah sounds good
